ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/PrivacyController.java

```diff
@@ -173,4 +178,38 @@ public long getSenderNonce(final Address sender, final String privacyGroupId) {
             // private state does not exist
             Account.DEFAULT_NONCE);
   }
+
+  private SendResponse sendRequest(final PrivateTransaction privateTransaction) {
+    final BytesValueRLPOutput bvrlp = new BytesValueRLPOutput();
+    privateTransaction.writeTo(bvrlp);
+    final String payload = BytesValues.asBase64String(bvrlp.encoded());
+
+    if (privateTransaction.getPrivacyGroupId().isPresent()) {
+      return enclave.sendBesu(
+          payload,
+          enclavePublicKey,
+          BytesValues.asBase64String(privateTransaction.getPrivacyGroupId().get()));
+    } else {
+      final List<String> privateFor =
+          privateTransaction.getPrivateFor().get().stream()
+              .map(BytesValues::asBase64String)
+              .collect(Collectors.toList());
+
+      if (privateFor.isEmpty()) {
+        privateFor.add(BytesValues.asBase64String(privateTransaction.getPrivateFrom()));
+      }
+      return enclave.sendLegacy(
```

Comment:
can sendLegacy just be "send" - the params should identify what's happening (then we can hide legacy/pantheon style from the API)